i18n

frodrigo opened the pull request:
Not to be merged.
Work in progress.
This branch is the a base for further work for Osmose-QA.

nlehuby commented on transport.validator.mapcss:
```diff
@@ -81,49 +81,49 @@ relation.pt_route[!to]
 node["public_transport"="platform"][!highway][!railway]
 {
-	throwError: "Jungle : Le tag historique est manquant (ajouter le tag highway=bus_stop/railway=tram_stop)";
+	throwError: tr("The historical tag is missing, add the tag highway=bus_stop / railway=tram_stop")
```
`legacy` (instead of `historical)` would be better

nlehuby commented on transport.validator.mapcss:
```diff
@@ -166,5 +166,5 @@ relation[type=route] > node[public_transport=platform]
 node[public_transport=platform]!.platform_ok
 {
-	throwError: "Jungle : L'arrêt n'est desservi par aucune ligne (l'ajouter à la relation route)";
+	throwError: tr("The stop is not served by any line, add it to a relation road");
```
`route relation`, not `relation road` ;)

frodrigo commented:
Thank you. fixed.

frodrigo commented:
Reintroduce French in po file

frodrigo commented:
Got the process to build the required zip for JOSM.

@nlehuby I let you ask for the support of it into JOSM. For the support in Osmose I will directly use the po file.

nlehuby commented:
Thanks a lot for that !

nlehuby commented:
According to Dirk, the mapcss should be in the root directory, not in the data.

And we need to find a way to publish the built zip too, with a stable url. 

Maybe an automated github release script can do the trick, but I'm not sure about the stable url.

frodrigo commented:
Ok, I simplified the process to build the zip file.

I think we can simply commit the zip file.

frodrigo commented:
I add the zip file to git to be available at download.

frodrigo commented:
Rebase this PR on my new "dev" branch.
This branch is now the base of my branch "osmose" https://github.com/frodrigo/transport_mapcss/tree/osmose.
I already use my branch "osmose" to update into Omose the "transport" analyser.
@nlehuby In Osmose I removed the two analyser in python you made and replaced it by code generated from the last mapcss of by my branch "osmose": https://github.com/frodrigo/osmose-backend/commit/188f8e4a8a0599fb3e39d4f3ae1becb440f9f69e

nlehuby commented on po/fr.po:
```diff
@@ -0,0 +1,119 @@
+msgstr "Le mode est manquant, ajouter un tag route = bus/coach/tram/etc"
+
+#: ../transport.validator.mapcss:26
+msgid "Missing transportation mode, chnage tag route to route_master"
```
`chnage` >> `change`

nlehuby commented on po/fr.po:
```diff
@@ -0,0 +1,119 @@
+#: ../transport.validator.mapcss:186
+msgid "The stop is not served by any line, add it to a route relation"
+msgstr ""
+"L'arrêt n'est desservi par aucune ligne (l''ajouter à la relation route)"
```
`la` >> `une` ?

frodrigo commented:
Ok. All done.

About Osmose: I delete python source code and move the item/class to the MapCSS generated code: https://github.com/frodrigo/osmose-backend/commit/188f8e4a8a0599fb3e39d4f3ae1becb440f9f69e#diff-511dd9d4ccd47d56b87c9b5ce8c32803L15

frodrigo commented:
I moved the already done translation from Osmose Pyhton code analyser here.
To help on translating this project we add an entry for it in Osmose Transifex: https://www.transifex.com/openstreetmap-france/osmose/transport_mapcss/

frodrigo commented:
OK for me as it is.
Github zip download not fit the need.

nlehuby commented:
Hum, there is something wrong, there is no data folder in the zip anymore :thinking:

frodrigo commented:
I removed because you tell me to do it, I probably misunderstood.
Which is the right layout ?

nlehuby commented:
I don't know ... let's skip to what works:
* a zip file named `transport_validator.zip`
* inside the zip, a `data` folder
* inside the data folder, the mapcss file and the lang files

There is also a regression, all the items within a group do not show up anymore ...
